code-challenges.test.js

```diff
@@ -54,6 +110,31 @@ var vowelTester2 = "academy"
 var vowelTester3 = "challenges"
 // Expected output: 2
 
-
-
 // b) Create the function that makes the test pass.
+//need a function that makes it equal to vowels and then returns at what index that first vowel is at
+
+// if (words.match(/[aeiou]/gi)){
+//     return words === null ? 0 : words.length
+//   }
+
+function vowels(words){
+    let vowelList = 'aeiouAEIOU';
+    let counter = 0;
+    let splitUp = words.split('')
+    for(let i=0; i<words.length; i++) {
+    if(vowelList.indexOf(words[i]) !== -1) {
+        console.log(words[i])
+        counter++
+    }
+  }
+ return counter;
```

Comment:
Good logic right here! 

I think you could refactor this to just return i when indexOf(words[i]) !== -1. 

Still great work